build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -3,17 +3,14 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
 
   <PropertyGroup>
+    <InitialOutDir>$(OutDir)</InitialOutDir>
     <NuGetOutDir>$(OutDir)Packages\</NuGetOutDir>
+    <LayoutDir>$(OutDir)PackagesLayout\</LayoutDir>
     <OutDir>$(OutDir)Raw\</OutDir>
+    <NuPkgOutput>$(NuGetOutDir)%(FileName).$(Version).nupkg</NuPkgOutput>
```

Comment:
This is confusing to me.  What is `%(FileName)` here?  There's no item we are referring to, right?